web/src/components/layout/user-menu.tsx

```diff
@@ -38,23 +40,31 @@ export default function UserMenu() {
         className={'user-menu ' + (showMenu ? 'active' : '')}
         onMouseEnter={() => setShowMenu(true)}
         onMouseLeave={() => setShowMenu(false)}>
+        <audio preload="auto" ref={audioRef} />
         <button
           className="toggle"
           title="click to play avatar"
-          onClick={() => {
+          onMouseEnter={() => {
             trackVoiceAvatar('self-listen', locale);
             if (account.avatar_clip_url !== null && !showAnimation) {
-              const audio = new Audio(account.avatar_clip_url);
-              audio.play();
+              audioRef.current.src = account.avatar_clip_url;
+
+              audioRef.current.play();
               setShowAnimation(!showAnimation);
-              audio.onended = () => {
+              audioRef.current.onended = () => {
```

Comment:
Why not attach these event listeners in the JSX?

Reply:
yeah, we can